buildconfig/download_win_prebuilt.py

```diff
@@ -117,10 +117,9 @@ def get_urls(x86=True, x64=True, sdl2=True):
 def download_prebuilts(temp_dir, x86=True, x64=True, sdl2=True):
     """ For downloading prebuilt dependencies.
     """
-    from distutils.dir_util import mkpath
     if not os.path.exists(temp_dir):
         print("Making dir :%s:" % temp_dir)
-        mkpath(temp_dir)
+        os.makedirs(temp_dir)
```

Comment:
There is a small behavioral change here, `mkpath` does not fail if directory exists, while `os.makedirs` can raise `FileExistsError` error. We need to do try-except handling for that (there is an `exist_ok` flag that can be passed to `os.makedirs` but that is not on py2, so we need to do the more verbose try-except handling)

Reply:
Good spot. But the os.makedirs is inside a check of whether the path exists, so it shouldn't be an issue. Also I think it's a bit funny that there's a `FileExistsError`. (Seems a bit niche).

Reply:
ohh yea my bad 😄 
I just saw the 2 line change diff and did not see the check for path exists